packages/sourcecred/src/core/ledger/grainIntegration.js

```diff
@@ -46,6 +49,7 @@ export type IntegrationConfig = {|
   // distributed funds via a configured integration
   processDistributions: boolean,
   currency: Currency,
+  integration: ?Object,
```

Comment:
This is a confusing line. Maybe it wants to be renamed `config` or something?

Reply:
This is confusing in the context of the named `IntegrationConfig` type, but I don't think its confusing once within the context of an integration. Within an integration, its pretty clearly just reading its own configuration.

From within the context of a grain integration, we have 
```
{
  accounting,
  currency,
  processDisitributions,
  integration
}
```

I'd be open to renaming it `self` but I think naming it `config` would more confusing and less descriptive in the alternative context, which needs to be much more approachable for devs looking to build their own integrations.

Reply:
I've added a comment to clarify what's going on instead.